### ISSUE TITLE
Task arrows show even if there is no note or comment content.

### ISSUE THREAD

**Issue by electricmessiah** [alex-LE/yourTinyTodo#77]
Tasks with no note/comment content have the left arrows to expand while other tasks without content display correctly without the arrow. In the old versions of MTT a task would only have the expansion arrow if there was a note attached.

**Comment by jeangld** [comment]
This is because the note field in ytt_todolist table is filled with NULL. If you manually update this to "", the arrows disappear. 

Personally, I don't like to work with NULL fields at all. One option is to change the database and change the field note to NOT NULL or fill the field null with "" in the code when inserting a new task.

**Comment by electricmessiah** [comment]
If that's the case, we should be able to check for a null in the field to make proper use of the dropdown arrow.

**Comment by alex-LE** [comment]
If we disable the arrow there will be no option to create a comment

**Comment by electricmessiah** [comment]
I thought the arrow denoted that there was content in the Note field. You can click the title of the entry to add notes and comments.

**Comment by alex-LE** [comment]
You are right, it makes sense. I will change this. I currently work on changing the theme to use bootstrap, so i can modify that also.

**Comment by electricmessiah** [comment]
I've been working with bootstrap a lot lately, so that's a welcome addition. ;)
